### PR TITLE
SSL Certificate command line parameter added 

### PR DESCRIPTION
**Pull request by aladagemre**
"List of SSL certificates separated by comma for frontend marathon_https_in"
"Ex: --ssl-certs /etc/ssl/mysite1_com.pem,/etc/ssl/mysite2_com.pem"

### CODE REVIEW

**Comment by aladagemre** [comment]
I added two fixes. First is SSL certificate command line parameter. It takes multiple ceritificate paths and generates the certificate paths.

    frontend marathon_https_in
      bind *:443 ssl crt /etc/haproxy/mysite_com.pem

Second one is a fix for machines without haproxy. There may be machines running haproxy on docker containers. This causes reload commands and verifications steps to fail (can't find haproxy command).

--skip-validation will skip the config validation.
And reload method will check if no haproxy exists on the machine. If so, it won't run a command for reload.

**Comment by brndnmtthws** [comment]
Looks good, but 3 things:
- Need to update the README
- Please write a unit test for this new functionality
- The [`run` script](https://github.com/mesosphere/marathon-lb/blob/master/run#L5-L13) needs to be update to handle the new parameter format

**Comment by aladagemre** [comment]
I have written unit tests for the ssl-certs argument. Updated run script and README.

File: README.md [hunk 1]
```diff
@@ -81,6 +81,33 @@ To get the full documentation run:
...
+You can provide your SSL certificate paths to be placed in frontend marathon_https_in section with `--ssl-certs`.
+
+``` console
+$ ./marathon_lb.py --marathon http://localhost:8080 --haproxy-config /etc/haproxy/haproxy.cfg --group external --ssl-certs /etc/ssl/site1.co,/etc/ssl/site2.co
+```
+
+If you are using the script directly, you have two options:
+
+ * Provide nothing and config will use `/etc/ssl/mesosphere.com.pem` as the certificate path. Put the certificate in this path or edit the file for the correct path.
+ * Provide --ssl-certs command line argument and config will use these paths.
```
Comment (brndnmtthws) [inline README.md]: Wrap with `--ssl-certs` for consistency.

File: README.md [hunk 2]
```diff
@@ -81,6 +81,33 @@ To get the full documentation run:
...
+
+If you are using run.sh or Docker image, you have three options:
+
+ * Provide your certificate text in HAPROXY_SSL_CERT environment variable. Contents will be written to `/etc/ssl/mesosphere.com.pem`. Config will use this path unless you specified extra certificate paths as in the next option.
```
Comment (brndnmtthws) [inline README.md]: Also wrap `HAPROXY_SSL_CERT`.

**Comment by brndnmtthws** [comment]
Some minor nits.

Otherwise, this is wonderful, thanks!

**Comment by brndnmtthws** [comment]
Closing in favour of #25

File: README.md (hunk 2 above)
Comment (skasturi) [inline README.md]: I am using HAPROXY_SSL_CERT. How to get this going? I pasted the text of both my certificates. But only the first one seems to be picked up. Can you please help? @brndnmtthws  Thank you!

Comment (skasturi) [inline README.md]: I am using DC/OS v 1.8.7 and Marathon LB docker image mesosphere/marathon-lb:v1.5.0

Comment (skasturi) [inline README.md]: @justinrlee @nlsun Can you please help here? (I am tagging you as I saw that you have the most recent commit). Apologies if incorrectly tagged. Hoping you can point me to the right person for this. Thank you.

Comment (skasturi) [inline README.md]: I already tried comma seperated strings of the ceritificates. But that doesn't seem to work.

**Pull request closed without merging**